apps/createsamples/utility.cpp

```diff
@@ -1402,7 +1402,7 @@ void cvCreateTestSamples( const char* infoname,
                 if( showsamples )
                 {
                     cvShowImage( "Image", &cvbgreader->src );
-                    if( cvWaitKey( 0 ) == 27 )
+                    if( (cvWaitKey( 0 ) & 0xFF) == 27 )
```

Comment:
@terfendail Decided to leave this here. Because the sample is mainly using C functionality, and since waitChar is not available as C version.